pages/common/tar.md

```diff
@@ -12,6 +12,10 @@
 
 `tar czf {{target.tar.gz}} {{file1}} {{file2}} {{file3}}`
 
+- Create a gzipped archive from directory omitting leading path:
```

Comment:
What is `omitting leading path` supposed to mean? It's pretty unclear here, so it should be changed.

Reply:
I've updated it, is it better now?